components/vc/shared/wheelSpeed/wheelSpeed.c

```diff
@@ -105,6 +112,26 @@ float32_t wheelSpeed_getSpeedLinear(wheel_E wheel)
     return mps;
 }
 
+float32_t wheelSpeed_getSlipRatio(void)
+{
+    return slipRatio;
+}
+
+static void calcSlipRatio_100Hz(void)
+{
+    float32_t FR_wheelSpeed = wheelSpeed_getSpeedLinear(WHEEL_FR);
+    float32_t RR_wheelSpeed = wheelSpeed_getSpeedLinear(WHEEL_RR);
```

Comment:
why only the right wheels? should you not average across the axle somehow too? on a right turn the right tires are slipping far less than the left, correct?